mysql-plugin/src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -30,11 +30,11 @@ public class TestSetupHooks {
 
   @Before(order = 1)
   public static void overrideUserAndPasswordIfProvided() {
-    String username = System.getenv("username");
+    String username = System.getenv("MYSQL_USERNAME");
```

Comment:
why `MYSQL`? This should be configurable right depending on the test??

Reply:
This file is inside mysql module so expecting MYSQL credentials only